rosidl_typesupport_connext_c/cmake/rosidl_typesupport_connext_c_generate_interfaces.cmake

```diff
@@ -251,8 +251,14 @@ if(NOT rosidl_generate_interfaces_SKIP_INSTALL)
     LIBRARY DESTINATION lib
     RUNTIME DESTINATION bin
   )
-endif()
 
-ament_export_libraries(${rosidl_generate_interfaces_TARGET}${_target_suffix} ${Connext_LIBRARIES})
+  ament_export_libraries(${rosidl_generate_interfaces_TARGET}${_target_suffix} ${Connext_LIBRARIES})
 
-ament_export_include_directories(include)
+  if(
+    NOT "${_generated_msg_files}${_generated_external_msg_files} " STREQUAL " " OR
+    NOT "${_generated_srv_files}${_generated_external_srv_files} " STREQUAL " "
+  )
+    ament_export_include_directories(include)
+  endif()
```

Comment:
I moved this block right after the install call for the generated msg and srv files.